GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -513,14 +513,13 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
             openFileWithToolStripMenuItem.Visible = isFile;
 
             toolStripSeparatorGitActions.Visible = isFile;
-            stopTrackingThisFileToolStripMenuItem.Visible = isFile;
-            stopTrackingThisFileToolStripMenuItem.Enabled = isExistingFileOrDirectory;
-            assumeUnchangedTheFileToolStripMenuItem.Visible = isFile;
-            assumeUnchangedTheFileToolStripMenuItem.Enabled = isExistingFileOrDirectory;
+            stopTrackingThisFileToolStripMenuItem.Visible = 
+                stopTrackingThisFileToolStripMenuItem.Enabled = isFile;
```

Comment:
Not good, too... If the file has been deleted or renamed, no need of this feature because that is done normally or more easily from the commit form.
You only want to use this feature if you want to keep the file (so don't see it in the commit form) and stop track it. So, you should keep the `isExistingFileOrDirectory` condition. So should be reverted.

Reply:
ok